test-apps/imodel-transformer/src/Transformer.ts

```diff
@@ -78,6 +80,31 @@ export class Transformer extends IModelTransformer {
       this._targetPhysicalModelId = PhysicalModel.insert(this.targetDb, IModel.rootSubjectId, "CombinedPhysicalModel"); // WIP: Id should be passed in, not inserted here
       this.importer.doNotUpdateElementIds.add(this._targetPhysicalModelId);
     }
+    if (options?.exportViewDefinition) {

```

Comment:
I don't suppose you currently have a way to filter geometry from geometry streams based on class (constructions etc)?

Reply:
Not yet, but that could be added. I think that would best be done in native code while cloning the GeometryStream (same spot as where entries are filtered by SubCategory). Seems like `Primary` should never be excluded. I can see wanting to exclude `Construction` and `Dimension`. Not sure about `Pattern`. Thoughts?

Reply:
Yes, patterns too. If view flags say not to draw some class of geometry ideally the transformer will exclude geometry of that class.

Reply:
To revisit in a future native PR.